Dockerfiles/base/00.python-redhat-ubi.Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM registry.access.redhat.com/ubi8/python-39
+FROM registry.access.redhat.com/ubi8-minimal
 LABEL maintainer="squad:git-defenders" url="https://github.ibm.com/whitewater/whitewater-detect-secrets"
 
 User root
-RUN yum -y update
+# install python 3.9 and corresponding pip ... install git (used in DS scan)
+RUN microdnf -y install python39 python39-pip git
```

Comment:
detect-secrets is currently designed to be compatible with up to Python 3.8: see https://github.com/IBM/detect-secrets/blob/master/.travis.yml#L21-L30. Although it does work with 3.9 as well, we don't have any official tests to support this version, so if something were to break, our CI / CD checks would not fail.

Would you mind installing Python 3.8 instead here? In a future update, we can add tests for 3.9.

Reply:
OK - using python 38